doc/content/introduction/installation/_index.md

```diff
@@ -136,7 +144,3 @@ processes are used that deploy build jobs to a number of build
 clients. In most scenarios, the master images need to be built on the
 machine where the software build is started.
 
-[^gopath_deprecation]: Older versions of the documentation contained references to $GOPATH
-    and recommendations to install programs into $GOPATH/bin. Since QMSTR 0.4,
-    GOPATH is not used anymore. The Makefiles instead expect that program
-    dependencies like _protoc_ to be found in the PATH.
```

Comment:
I think we should keep it. What do you think @GiasemiSh ?

Reply:
Yes true, we should keep this comment.

Reply:
One question is why do we have the GOPATH as variable in our Makefile if we don't need it? Probably should check if it's safe to remove. If not, this comment is misleading.